test/test_checks.py

```diff
@@ -11,6 +11,10 @@ def test_get_noqa_lines(self):
 
 
 class TestFlake8Stdin(TestCase):
+    def setUp(self):
+        if not hasattr(self, 'assertRegex'):
+            self.assertRegex = self.assertRegexpMatches
+
```

Comment:
According to your PR, `assertRegexpMatches` has been deprecated since Python@3.2, which has been out since 2011. No need to worry about Python@2. We can safely drop this fallback =)

https://github.com/python/cpython/pull/28268/files#diff-14c3c1d897d1e85767303f24626acaa6d1ea1162219c72caa240ef9f533ebe99L1636-L1638

https://www.python.org/download/releases/3.2/

```suggestion
```